flavours/twofa/manifest.fragment.json

```diff
@@ -9,7 +9,6 @@
       "type": "password"
     },
     "two_fa_code": {
-      "type": "hidden",
```

Comment:
Let's keep it hidden since we don't use it

Reply:
`hidden` is not handled by Harvest, it's replaced by a text input. So specifying `hidden` here does not do anything.

Reply:
Ok then